stacker_blueprints/aws_lambda.py

```diff
@@ -219,3 +223,44 @@ def create_template(self):
         self.create_role()
         self.create_function()
         self.create_policy()
+
+
+class FunctionScheduler(Blueprint):
+
+    VARIABLES = {
+        "CloudwatchEventsRule": {
+            "type": TroposphereType(events.Rule),
+            "description": "The troposphere.events.Rule object params.",
+        },
+    }
+
+    def create_scheduler(self):
+        variables = self.get_variables()
+        troposhere_events_rule = variables["CloudwatchEventsRule"]
+        aws_lambda_arns = []
```

Comment:
I'd change this into a dictionary - where the keys are the `Target.Id` and the values are the Arns

Reply:
You'll also likely want to run the Id through https://github.com/remind101/stacker/blob/master/stacker/util.py#L273